extensions/kind/src/create-cluster.ts

```diff
@@ -29,6 +29,7 @@ import { parseAllDocuments, stringify } from 'yaml';
 
 import ingressManifests from '/@/resources/contour.yaml?raw';
 
+import { getFreePort } from '../../../packages/main/src/plugin/util/port';
```

Comment:
@benoitf is there a better way to reuse port.ts? Should I use available npm package instead?

Reply:
Maybe you could add it using a separate PR to `packages/main/src/plugin/extension/extension-loader.ts`, and the use the `extensionApi` to call it?